tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Introduce the DW acronym for unfamiliar readers ?</p><p> </p><p>... domain-wall (DW) ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>